docs/go-bitswap.puml

```diff
@@ -3,15 +3,17 @@
 node "Top Level Interface" {
```

Comment:
Is there a script that generates the image out of this puml file?

Reply:
There isn't, but it's easy to find generators online (I just searched for "online puml generator")